be/src/common/config.h

```diff
@@ -256,21 +222,18 @@ CONF_Int32(file_descriptor_cache_capacity, "32768");
 // modify them upon necessity
 CONF_Int32(min_file_descriptor_number, "60000");
 CONF_Int64(index_stream_cache_capacity, "10737418240");
-// CONF_Int64(max_packed_row_block_size, "20971520");
 
 // Cache for storage page size
 CONF_String(storage_page_cache_limit, "20%");
 // whether to disable page cache feature in storage
-CONF_Bool(disable_storage_page_cache, "false");
+CONF_mBool(disable_storage_page_cache, "false");
```

Comment:
There is a case that `disable_storage_page_cache ` is true at first, then changed to false, will the memory used by cache has no chance to free?

Reply:
The config `disable_storage_page_cache`is not used to close cache but to choose whether to use the cache during read.  So the memory used by cache will not be freed.